src/mixins/index.ts

```diff
@@ -9,3 +10,4 @@ export { default as PeopleRolesMixin } from '@/mixins/people-roles-mixin'
 
 // external mixin
 export { NameRequestMixin } from '@bcrs-shared-components/mixins'
+export { CountriesProvincesMixin } from '@bcrs-shared-components/mixins'
```

Comment:
When I try to import twice from the same source, VS Code gives me an error ...?

I recommend combining lines 12 and 13.

Reply:
Oops....